lib/teamlab/response.rb

```diff
@@ -40,6 +40,10 @@ def data
       @body['response']
     end
 
+    def self.response_is_correct?(response)
+      response.success && response.error.nil? && response.body.is_a?(Hash)
```

Comment:
Two things

1. Add more detailed error if any of this data is incorrect

For example is responce.success is not true - this method return just false, but I like a idea then there will be Exception that `Response should be always successfull`. This help to simplify debugging and make logs more readable

2. I think this should be not `self.` method, but just simple `def correct?`

And called like `expect(RequestHelper.current_responce).to be_correct`

Reply:
1. Added in [commit](https://github.com/ONLYOFFICE/onlyoffice_api_gem/pull/303/commits/11644e3d0dabcc130836d3cfe61f4cb871f7d49c)
2. Doesn't work without `self`